test/size.txt

```diff
@@ -1,62 +1,62 @@
-170.87 kB   |    15.49 kB   |     6.01 kB   |   v0/amp-form-0.1.js
- 41.05 kB   |     4.33 kB   |      1.9 kB   |   v0/amp-fresh-0.1.js
- 51.95 kB   |     7.08 kB   |     2.88 kB   |   v0/amp-fx-flying-carpet-0.1.js
- 90.88 kB   |     6.07 kB   |     2.56 kB   |   v0/amp-gfycat-0.1.js
-119.77 kB   |     8.38 kB   |     3.46 kB   |   v0/amp-google-vrview-image-0.1.js
- 174.2 kB   |    15.21 kB   |     6.07 kB   |   v0/amp-iframe-0.1.js
- 245.8 kB   |    32.16 kB   |    10.62 kB   |   v0/amp-image-lightbox-0.1.js
-117.47 kB   |     7.24 kB   |     3.02 kB   |   v0/amp-instagram-0.1.js
-101.79 kB   |      8.5 kB   |      3.6 kB   |   v0/amp-install-serviceworker-0.1.js
- 98.09 kB   |     7.03 kB   |     2.93 kB   |   v0/amp-jwplayer-0.1.js
-135.01 kB   |     8.21 kB   |     3.36 kB   |   v0/amp-kaltura-player-0.1.js
-153.04 kB   |    12.51 kB   |     4.74 kB   |   v0/amp-lightbox-0.1.js
-146.43 kB   |    13.62 kB   |     4.79 kB   |   v0/amp-lightbox-viewer-0.1.js
-102.68 kB   |     5.91 kB   |     2.54 kB   |   v0/amp-list-0.1.js
-164.77 kB   |    14.21 kB   |     5.35 kB   |   v0/amp-live-list-0.1.js
-155.67 kB   |    40.33 kB   |    14.43 kB   |   v0/amp-mustache-0.1.js
- 91.92 kB   |      6.7 kB   |     2.72 kB   |   v0/amp-o2-player-0.1.js
-145.74 kB   |    20.36 kB   |     6.09 kB   |   v0/amp-pinterest-0.1.js
- 90.51 kB   |     5.97 kB   |     2.52 kB   |   v0/amp-reach-player-0.1.js
-100.74 kB   |     5.83 kB   |     2.52 kB   |   v0/amp-share-tracking-0.1.js
-104.44 kB   |    10.04 kB   |     3.77 kB   |   v0/amp-sidebar-0.1.js
-118.24 kB   |     9.81 kB   |     3.92 kB   |   v0/amp-slides-0.1.js
-130.86 kB   |    14.25 kB   |     5.41 kB   |   v0/amp-social-share-0.1.js
- 91.52 kB   |     6.29 kB   |     2.63 kB   |   v0/amp-soundcloud-0.1.js
- 98.98 kB   |     7.76 kB   |     3.04 kB   |   v0/amp-springboard-player-0.1.js
-   115 kB   |     8.05 kB   |     3.24 kB   |   v0/amp-sticky-ad-0.1.js
-156.32 kB   |    15.62 kB   |     6.29 kB   |   v0/amp-twitter-0.1.js
-132.43 kB   |    10.41 kB   |     4.06 kB   |   v0/amp-user-notification-0.1.js
- 90.94 kB   |     6.07 kB   |     2.57 kB   |   v0/amp-vimeo-0.1.js
- 90.54 kB   |     5.93 kB   |     2.53 kB   |   v0/amp-vine-0.1.js
-643.43 kB   |   514.29 kB   |   168.53 kB   |   v0/amp-viz-vega-0.1.js
- 142.2 kB   |     8.74 kB   |      3.6 kB   |   v0/amp-youtube-0.1.js
- 12.87 kB   |     3.58 kB   |     1.58 kB   |   v0/cache-service-worker-0.1.js
-207.07 kB   |    45.93 kB   |    15.98 kB   |   current-min/f.js / current/integration.js
\ No newline at end of file
+      max   |         min   |             gzip   |   file
+      ---   |         ---   |              ---   |   ---
+  70.5 kB   |     5.62 kB   |          2.45 kB   |   alp.js / alp.max.js
+221.74 kB   |     61.3 kB   |   check-types.js
+774.03 kB   |   143.97 kB   |         45.46 kB   |   shadow-v0.js / amp-shadow.js
+    428 B   |       278 B   |       sw-kill.js
+    537 B   |       382 B   |            sw.js
```

Comment:
oh i mean the filename is in the wrong column. this happens if it cant find the min/max counterpart.